docs/miscellaneous.rst

```diff
@@ -56,6 +56,22 @@ So for the following contract snippet::
 
 The position of ``data[4][9].b`` is at ``sha3(uint256(9) . sha3(uint256(4) . uint256(1))) + 1``.
 
+********************
+When Solidity Throws
+********************
+
+A Solidity contract throws an exception for unhealthy operations such as
+
+- division by zero
+- modulo by zero
+- out-of-bounds index access on an array
+- out-of-bounds index access on a fixed length bytes
```

Comment:
Maybe change it to "fixed-length ``bytes``" so the reference to the type is clearer